homeassistant/components/sensor/postnl.py

```diff
@@ -24,14 +24,15 @@
 
 DEFAULT_NAME = 'postnl'
 
-ICON = 'mdi:package-variant-closed'
-
 MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=30)
 
+CONF_LETTER = 'letter'
```

Comment:
Because it would require a definition in `const.py`. If only one platform is using it then it doesn't make much sense to add a constant to that file..